protocol-configuration/mainnetAddresses.ts

```diff
@@ -1096,6 +1123,12 @@ export const MainnetContractsConfig = {
     address: '0x34809aEDF93066b49F638562c42A9751eDb36DF5',
     category: AddressCategory.External
   },
+  lusdToDaiLBPPool: {
+    // poolId: 0x8485b36623632ffa5e486008df4d0b6d363defdb00020000000000000000034a
+    artifactName: 'IWeightedPool',
+    address: '0x8485b36623632ffa5e486008df4d0b6d363defdb',
```

Comment:
@eswak this contract is not verified on etherscan

Reply:
Thats the balancer LBP it came from the factory and we don't have the tooling to verify but balancer team does